### PR TITLE
Add react-native-reanimated config

### PR DESCRIPTION
**Pull request by friederbluemle**
Closes #197

### CODE REVIEW

**Comment by LcTwisk** [comment]
Hey! Is there anything I can do to help getting this merged?

File: plugins/ern_v0.14.0+/react-native-reanimated_v1.2.0+/config.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "root": "android",
+    "dependencies": [
+      "com.android.support:transition:28.0.0"
+    ]
+  },
+  "ios": {
+    "copy": [
+      {
+        "dest": "{{{projectName}}}/Libraries/RNReanimated",
+        "source": "ios/**"
```
[the remark this answers is not in the document]
Comment (LcTwisk) [reply plugins/ern_v0.14.0+/react-native-reanimated_v1.2.0+/config.json:12]: I had to change the Xcode project path to make it work: https://github.com/LcTwisk/electrode-native-manifest/commit/e55fcaf9fc466ae67fee61de14ae54d1f53165a9

For some reason there were two project files copied to the container (everything was copied twice). Probably, there’s a better way to fix it but didn’t have time to dive into the code.

[the remark this answers is not in the document]
Comment (friederbluemle) [reply plugins/ern_v0.14.0+/react-native-reanimated_v1.2.0+/config.json:12]: Thanks @LcTwisk! Yes, I _think_ that's the reason we should use a single star here

File: plugins/ern_v0.14.0+/react-native-reanimated_v1.3.0+/config.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": [
+      "androidx.transition:transition:1.1.0"
```
Comment (LcTwisk) [inline plugins/ern_v0.14.0+/react-native-reanimated_v1.3.0+/config.json:5]: @friederbluemle React Native 0.60.x was the first version using Android X,  should this plugin be moved to the `ern_v0.39.0+` folder, or doesn't it work like that?

Comment (friederbluemle) [reply plugins/ern_v0.14.0+/react-native-reanimated_v1.3.0+/config.json:5]: Until now (or recently), there has only been a _very loose coupling_ between version of React Native and ERN. So while there is a default when a new project is created, it should also be possible to use a different combination of RN/ERN for existing projects. There is some internal discussion about consolidating these plugins and removing the need to put them into different folders in the future. Until then I think we should keep the plugin configs at the lowest supported version, and only move them "up" if they specifically contain a config directive that was added in a newer version of ERN. Hope that makes sense.

Comment (LcTwisk) [reply plugins/ern_v0.14.0+/react-native-reanimated_v1.3.0+/config.json:5]: Got it. Ideally, there would be 2 versions of this plugin: One with a dependency on the support library for < RN 0.60. And on with the Android X dependency for >= RN 0.60. 
Anyway, I understand that would make it even more complicated as you will have to introduce plugins per RN version for each ERN version 😅

**Pull request merged**